backend/LoadCell.py

```diff
@@ -0,0 +1,104 @@
+from enum import Enum
```

Comment:
I see that not all the backend files have this right now but I would like for it to be standard to include a file header along with region identifiers like seen in this file: https://github.com/StudentOrganisationForAerospaceResearch/RocketControlUnit/blob/master/backend/src/ThreadManager.py

by region identifiers I mean like:
`# General imports =================================================================================`

Reply:
added headers and region identifiers as requested